turbopack/crates/turbo-persistence/src/static_sorted_file.rs

```diff
@@ -397,6 +402,12 @@ impl StaticSortedFile {
                 self.meta.blocks_start()
             );
         }
+        #[cfg(unix)]
+        let _ = self.mmap.advise_range(
+            memmap2::Advice::WillNeed,
```

Comment:
```suggestion
            memmap2::Advice::Sequential,
```

> Expect page references in sequential order. (Hence, pages in the given range can be aggressively read ahead, and may be freed soon after they are accessed.)

-- https://docs.rs/memmap2/latest/memmap2/enum.Advice.html#variant.Sequential

> If your access is sequential, you should use MADV_SEQUENTAL -- it should do the read-ahead and then drop the pages once you have accessed them giving you optimal performance and memory management.

-- https://stackoverflow.com/a/31214471